packages/shared/providers.yaml

```diff
@@ -431,6 +434,9 @@ asana-scim:
     auth_mode: API_KEY
     proxy:
         base_url: https://app.asana.com/api
+        verification:
```

Comment:
This wouldn't work as a POST would require a body and this API endpoint looks incorrect for SCIM https://developers.asana.com/docs/scim#user-endpoints